_data/segwitsupport.csv

```diff
@@ -155,3 +155,4 @@ Yogh,http://yogh.io,ready,block explorer,bitcoin core
 MiningRigRentals,https://www.miningrigrentals.com.com,ready,mining pool/rental services,n/a
 HitBTC, https://hitbtc.com, ready, exchange,
 Colibit, https://www.colibit.io, ready, exchange
+Bitfortip, https://www.bitfortip.com/, ready, social reward network. Tip Bitcoins as an incentive | Earn Bitcoins for being helpful.
```

Comment:
Can you reduce the node to a minimum (this is not an advertising text field).

Reply:
Done